packages/ibm-products/src/components/Card/Card.tsx

```diff
@@ -35,7 +40,7 @@ type LinkType = {
   [key: string]: unknown;
 };
 
-export interface ActionIcon extends Metadata {
+export interface ActionIcon extends Metadata, HTMLAttributes<HTMLElement> {
```

Comment:
Something like this might be better:
```
[key: `data-${string}`]: string;
```

Just to scope what is acceptable to pass to `ActionIcon`, it likely doesn't need to extend every HTMLAttribute from `HTMLElement`.